bundles/org.dataflowanalysis.analysis.dfd/src/org/dataflowanalysis/analysis/dfd/core/DFDFlowGraphCollection.java

```diff
@@ -57,6 +58,16 @@ public List<? extends AbstractTransposeFlowGraph> findTransposeFlowGraphs() {
             logger.error("Cannot find transpose flow graphs for non-dfd resource provider");
             throw new IllegalArgumentException();
         }
-        return new DFDTransposeFlowGraphFinder(dfdResourceProvider).findTransposeFlowGraphs();
+        
+        var transposeFlowGraph = transposeFlowGraphFinder.findTransposeFlowGraphs();
```

Comment:
direclty return `transposeFlowGraphFinder.findTransposeFlowGraphs();`. There is no need for an extra variable

Reply:
needed for earlier implementation of has cycles, removed now